docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -360,7 +360,7 @@ The configuration above generates the following sidebar:
 
 ## Internationalization
 
-Use the `translations` property on link and group entries to translate the link or group label for each supported language.
+Use the `translations` property on link and group entries to translate the link or group label for each supported language by specifying a BCP-47 language tag, e.g. `"en"`, `"ar"`, or `"zh-CN"`, as the key and the translated label as the value.
```

Comment:
It feels like that if we link to something, it should be a complete list and this one is only a small subset of commonly used primary language subtags (they even say less than 2 percent of primary language subtags). 

Considering the [official registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) is not the "most user-friendly", maybe we could link to https://r12a.github.io/app-subtags/ ?

Any thoughts? Or maybe the wikipedia one is enough?

Reply:
That’s the app I use regularly, but has the drawback of not providing much context. https://www.w3.org/International/questions/qa-choosing-language-tags might be more handy if you’re looking to learn what BCP-47 means?

Reply:
Great point! As a user following this guide, I am thinking about learning more about what BPC-47 means. However, the following line explains it very well with the example. 

So it would make sense to share a website where you can enter a language and you can find out the BPC-47 for that language.

I think the github page link would be best.

Reply:
> [w3.org/International/questions/qa-choosing-language-tags](https://www.w3.org/International/questions/qa-choosing-language-tags) might be more handy if you’re looking to learn what BCP-47 means?

Great page, didn't know about this one. I also just noticed that https://r12a.github.io/app-subtags/ is actually linked in it as the "Language Subtag Lookup tool" so considering this, I think I may slightly prefer the W3C one for a guide too. 🤷

Reply:
Feel free to choose what you think is best @HiDeoo!

Reply:
Went with the W3C one as it also links to the other one.

Thanks Kevin for the suggestion of adding a link and Chris for that link 🙌 If I got that right, everyone should get their co-author credits.